packages/Radio/stories/examples/Grouping.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { boolean, select, text } from "@storybook/addon-knobs";
+import { Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import Heading from "@paprika/heading";
+import { RadioStory } from "../Radio.stories.styles";
+import Radio from "../../src/Radio";
+
+const radioProps = () => ({
+  size: select("size", ShirtSizes.DEFAULT, "medium"),
+  isDisabled: boolean("isDisabled", false),
+  isChecked: boolean("isChecked", false),
+  canDeselect: boolean("canDeselect", false),
+  a11yText: text("a11yText", ""),
+});
+
+const ExampleStory = props => {
```

Comment:
To me sound that this Group radio button should be a uncontrolled component, my concern is that is too much boiler plate when what a consumer pretty much wants it o know which component is checked.

I would approach this API like the following:
```jsx
<Radio.Group 
   defaultCheck={(check) => check.value.id === "something"} 
   onChange={(value) => { console.log(value) }}
>
   <Radio value={whatever}>First</Radio>
   <Radio value={whatever}>second</Radio>
   <Radio value={whatever}>third</Radio>
<Radio.Group>
```

This achieve two things one removes the need of any state from the consumer and give the ability to set the initial value via a function that can be used to check which checkbox should be initially selected. 

what do you think? @mikrotron @tristanjasper

Reply:
Also this way we are more align the way that most input component works like having a onChange value. like the input, select and pretty much any react component.

Reply:
Sure that is another option, I guess there isn't really a case for a single radio component to be needed, ie without the surrounding radioGroup container so yeah would be cleaner. I'll move forward with this suggestion

Reply:
> Also this way we are more align the way that most input component works like having a onChange value. like the input, select and pretty much any react component.

Agreed